rioting-daisies/BangazonAPI
# TrainingProgram Controller and testing

# Description

trainingProgram-
created controller for training program and created test for it
programs should include a list of employees if they signed up for the program
can put ?completed=false or true to show programs that have finished or not




Fixes Issue #  (https://github.com/rioting-daisies/BangazonAPI/issues/8)

## Type of change


# Testing Instructions

git fetch
git checkout at-trainingProgram

make sure all tests work
run api and make sure you can see trainingprograms with employees listed
able to use the completed in the URL



# Checklist:
